packages/README.md

```diff
@@ -46,6 +46,7 @@ When creating a new package, you need to provide at least the following:
     - Installation details
     - Usage example
     - API documentation, if applicable ([more info](#maintaining-api-documentation))
+    - A link to the contributing guidelines
```

Comment:
Should we put the full text of the new section proposed so people can copy and paste it?

Reply:
Maybe something like a script to auto-generate a new package would be better? Not something should be covered in this PR though but just an idea 🙈 .

Reply:
I've added a link to the text in one of the packages. 👍